packages/tests/integration/src/EntityManager.txns.test.ts

```diff
@@ -1,171 +0,0 @@
-import { Stepper } from "@src/Stepper.test";
-import { Publisher, SmallPublisher } from "@src/entities";
-import { select } from "@src/entities/inserts";
-import { newPgConnectionConfig } from "joist-orm";
-import { Pool } from "pg";
-
-import { knex, newEntityManager } from "@src/testEm";
-
-describe("EntityManager", () => {
```

Comment:
Seems like this entire test file was to test anomalies based on serializable isolation level or not